modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2258,6 +2234,30 @@ private void _testAddObjectDefinition(boolean modifiable, boolean system)
 		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition);
 	}
 
+	private void _testBindObjectDefinitions(
+			Map<String, String[]> expectedMap,
+			List<ObjectRelationship> objectRelationships,
+			long rootObjectDefinitionId)
+		throws Exception {
+
+		TreeTestUtil.bind(_objectDefinitionLocalService, objectRelationships);
+
+		for (ObjectRelationship objectRelationship : objectRelationships) {
+			Assert.assertEquals(
+				objectRelationship.getDeletionType(),
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE);
```

Comment:
@murilostodolni you need to perform a get request because those objectRelationships are already cascade.

I suggest you create a new method in ObjectRelationshipTestUtil to create a new relationship passing the desired deletion type.

Reply:
Done